### DISCUSSION TITLE
Any alternatives for file selection window in ComfyUI?

### DISCUSSION THREAD

**Discussion by okolenmi** [Comfy-Org/ComfyUI#3514] [Q&A] [answered]
Really, when you have a lot of models/lora/etc. it's very difficult to find something as all files presented as a single list. Yes, I can find files by folder name if I can remember it, but if I just want to see what I have it's just a nightmare list... Is there any alternatives to browse file folders to choose a file? Such an extension will be very helpfull, I think.

**Comment by andreszs** [comment, marked as answer]
Regarding lora selection, I'm using the **Power Lora Loader** node from the rgthree extensions, which organizes them in folders.

![image](https://github.com/comfyanonymous/ComfyUI/assets/10298071/33f504b7-a90f-44fd-857f-243833a3ae97)

I agree that it would be great if ComfyUI supported this arrangement by default for all file selectors.

**Reply by okolenmi** [reply]
It looks like new list isn't from this extension. I have the default long list.

**Reply by okolenmi** [reply]
Right click on ComfyUI workflow -> rgthree comfy -> settings, and then after activating checkbox it works for everything (models, lora, images etc.) Thank you.
![изображение](https://github.com/comfyanonymous/ComfyUI/assets/55319814/f1494096-6323-4679-81d1-57a707963f1a)
